.github/workflows/run_torchbench.yml

```diff
@@ -70,7 +74,8 @@ jobs:
           # shellcheck disable=SC1091
           . "${HOME}"/anaconda3/etc/profile.d/conda.sh
           conda activate pr-ci
-          python3 pytorch/.github/scripts/run_torchbench.py \
+          # python3 -c "import torch; torch.rand((4, 4), device='cuda')"
```

Comment:
Looks like we can remove this? I am also okay to keep and uncomment it, just to make sure the hardware works.

Reply:
thanks for catching that - I will remove it. It doesn't actually work because pytorch isn't built at this point.